timemachine/potentials/bonded.py

```diff
@@ -115,6 +115,9 @@ def harmonic_bond(conf, params, box, lamb, bond_idxs, lamb_mult=None, lamb_offse
     * lamb argument is unused
 
     """
+    if bond_idxs.shape[0] == 0:
+        return 0.0
+
     assert params.shape == bond_idxs.shape
```

Comment:
Suggestion: Is it preferable to move this one to be right after the shape assertion?

Reply:
Fixed in [c6ea7e9](https://github.com/proteneer/timemachine/pull/719/commits/c6ea7e9b6810816ba18f44a9b4b1f5f8eb95822b)